sdk/python/feast/repo_config.py

```diff
@@ -25,7 +26,64 @@ class RepoConfig(NamedTuple):
     online_store: OnlineStoreConfig
 
 
+# This is the JSON Schema for config validation. We use this to have nice detailed error messages
+# for config validation, something that bindr unfortunately doesn't provide out of the box.
+#
+# The schema should match the namedtuple structure above. It could technically even be inferred from
+# the types above automatically; but for now we choose a more tedious but less magic path of
+# providing the schema manually.
+
+config_schema = {
+    "type": "object",
+    "properties": {
+        "project": {"type": "string"},
```

Comment:
This json schema is arguably going to be quite important for people to understand how they can configure Feast. Can we maybe 
1. Add descriptions to the fields?
2. Move it out to a top level file (`config_schema.json`?)